_config.yml

```diff
@@ -92,47 +94,45 @@ paginate_path: "blog/page:num"
 markdown: kramdown
 kramdown:
   parse_block_html: true
-permalink: /blog/:year/:month/:title/
-highlight: rouge
-plugins_dir: [jekyll-paginate, jekyll-redirect-from]
+permalink: /blog/:year/:month/:title.html
+highlight: pygments
 excerpt_separator: "<!--more-->"
-include: ['.htaccess']
-# Exclude these files from your production _site
+include:
+  - .htaccess
 exclude:
-    - .sass-cache
-    - Gemfile
-    - Gemfile.lock
-    - LICENSE
-    - README.md
-    - INSTALL.md
-    - vendor/
+  - .sass-cache
+  - Gemfile
+  - Gemfile.lock
+  - LICENSE
+  - README.md
+  - INSTALL.md
+  - Makefile
+  - vendor/
+
+# Build settings
+# - https://github.com/jekyll/jekyll-redirect-from
 
 plugins:
-  - jekyll-get-json
+  - jekyll-paginate
+  - jekyll-redirect-from
+   -jekyll-get-json
 
 ## external data sources
 jekyll_get_json:
   - data: all_badged_people
     json: 'https://feeds.carpentries.org/all_badged_people.json'
-  - data: all_past_workshops
-    json: 'https://feeds.carpentries.org/all_past_workshops.json'
-  - data: all_upcoming_workshops
-    json: 'https://feeds.carpentries.org/all_upcoming_workshops.json'
   - data: badges_stats
     json: 'https://feeds.carpentries.org/badges_stats.json'
-#  - data: community_lessons
-#    json: 'https://feeds.carpentries.org/community_lessons.json'
+  - data: community_lessons
```

Comment:
At the moment this is a manually curated file (but that will change with #25) so this needs to remain commented out or we will will in all the Carpentries Incubator information